src/gtk2.mk

```diff
@@ -9,7 +9,7 @@ $(PKG)_CHECKSUM := ac2ac757f5942d318a311a54b0c80b5ef295f299c2a73c632f6bfb1ff49cc
 $(PKG)_SUBDIR   := gtk+-$($(PKG)_VERSION)
 $(PKG)_FILE     := gtk+-$($(PKG)_VERSION).tar.xz
 $(PKG)_URL      := https://download.gnome.org/sources/gtk+/$(call SHORT_PKG_VERSION,$(PKG))/$($(PKG)_FILE)
-$(PKG)_DEPS     := cc atk cairo gdk-pixbuf gettext glib jasper jpeg libpng pango tiff
+$(PKG)_DEPS     := cc atk cairo gdk-pixbuf gettext glib jasper jpeg libpng pango tiff libwusers
```

Comment:
What makes GTK+ 2 need `libwusers`? There's no extra patch (yet?) and it built without it, so I'm just curious

Reply:
It detects `pwd.h` and `grp.h` and needs a library to use the user database API they declare.

Another way to deal with that can be `pragma comment(lib, "implib-name.lib"`, of course. It would tie `libwusers` to a Microsoft specific pragma opportunistically supported by CLang, though.

Reply:
Aaah, gotcha.